scripts/debug_tools/prepare_all_cmd_for_ctu.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import argparse
+import os
+import subprocess
+
+
+def execute(cmd):
+    print("Executing command: " + ' '.join(cmd))
+    try:
+        proc = subprocess.Popen(cmd,
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
+        out, err = proc.communicate()
+
+        print("stdout:\n\n" + out.decode("utf-8"))
+        print("stderr:\n\n" + err.decode("utf-8"))
+
+        if proc.returncode != 0:
+            print('Unsuccessful run: "' + ' '.join(cmd) + '"')
+            raise Exception("Unsuccessful run of command.")
+        return out
+    except OSError:
+        print('Failed to run: "' + ' '.join(cmd) + '"')
+        raise
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Prepare all commands '
+        'to execute in local environmennt for debugging.')
+    parser.add_argument('--clang', required=True)
+    parser.add_argument('--clang_plugin_name', default=None)
+    parser.add_argument('--clang_plugin_path', default=None)
+    args = parser.parse_args()
+
+    if ("reports/failed" not in str(os.getcwd()) or
+        os.path.basename(os.path.normpath(os.getcwd())) !=
+            "failed"):
+        print("Please cd to .../reports/failed")
+        exit(-1)
+
+    tools_dir = os.path.dirname(os.path.realpath(__file__))
+
+    out = execute([os.path.join(tools_dir, "prepare_compile_cmd.py"),
```

Comment:
Maybe instead of executing other python scripts using this method you could just import them and make a function call. I think this would be a cleaner solution.